mhkit/utils/stat_utils.py

```diff
@@ -1,9 +1,57 @@
-from mhkit import qc
+"""
+This module contains functions to perform various statistical calculations 
+on continuous data. It includes functions for calculating statistics such as
+mean, max, min, and standard deviation over specific windows, as well as functions
+for vector/directional statistics. The module also provides utility functions 
+to unwrap vectors, compute magnitudes and phases in 2D/3D, and calculate 
+the root mean squared values of vector components.
+
+Functions:
+----------
+- get_statistics: Calculates statistics for continuous data.
+- vector_statistics: Calculates vector mean and standard deviation.
+- unwrap_vector: Unwraps vector data to fall within a 0-360 degree range.
+- magnitude_phase: Computes magnitude and phase for 2D or 3D data.
+- unorm: Computes root mean squared value of 3D vectors.
+"""
+
 import pandas as pd
 import numpy as np
+from mhkit import qc
+
+
+def _calculate_statistics(datachunk, vector_channels):
```

Comment:
New function called by `get_statistics` to reduce the number of local variables
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-locals